dev-docs/plugins/bc/about-bc-prebid-plugin.md

```diff
@@ -126,9 +137,19 @@ The MailOnline plugin is used by the prebid plugin to render the ad. This plugin
 - Repository location:  `https://github.com/prebid/videojs-mailonline-plugin.git`
    - after building: `./videojs-mailonline-plugin/dist/videojs_5.vast.vpaid.js`
 
+### Brightcove IMA Plugin
+
+The Brightcove IMA plugin can be used by the prebid plugin to render the ad, depending on the prebid config options defined. By default, if the prebid config options identifies DFP as the primary ad server, then the Brightcove IMA Plugin is used to render the ad.  The publisher can also use the `adRenderer` option to specify that the IMA plugin is used to render the ad, regardless of the presence of the DFP parameters.
+
+{% capture infoNote %}
+If your page includes more than one Brightcove Player within the same HTML document and *any* of these players loads the Brightcove IMA Plugin, then it is recommended that *all* of your players in the document use the Brightcove IMA Plugin. Therefore, even if your prebid configuration does not include DFP parameters, in this case you should explicitly specify the IMA plugin using the `adRenderer` option.
+{% endcapture %}
+
+{% include alerts/alert_note.html content=infoNote %}
+
```

Comment:
Let's hold off on doing this modification to the doc.  We may need to change the wording of it depending on the outcome of our testing here

Reply:
Let's hold off on doing this modification to the doc.  We may need to change the wording of it depending on the outcome of our testing here

Reply:
Let's go ahead and add the infoNote but use this wording instead:

If you will be using the Brightcove IMA plugin as the ad renderer, either because DFP is the primary ad server or you have explicitly specified the IMA plugin in the `adRenderer` option, then it is SUGGESTED that you add the IMA plugin to your Brightcove Player configuration in the Brightcove Studio.  Doing so prevents some problems observed when the Player is running on iOS and the IMA plugin is loaded at run-time.